blivet/formats/lvmpv.py

```diff
@@ -96,55 +95,29 @@ def dict(self):
                   "peStart": self.peStart, "dataAlignment": self.dataAlignment})
         return d
 
-    def create(self, **kwargs):
-        """ Write the formatting to the specified block device.
-
-            :keyword device: path to device node
-            :type device: str
-            :raises: FormatCreateError
-            :returns: None.
-
-            .. :note::
-
-                If a device node path is passed to this method it will overwrite
-                any previously set value of this instance's "device" attribute.
-        """
+    def _create(self, **kwargs):
         log_method_call(self, device=self.device,
                         type=self.type, status=self.status)
 
-        try:
-            DeviceFormat.create(self, **kwargs)
-            # Consider use of -Z|--zero
-            # -f|--force or -y|--yes may be required
+        # Consider use of -Z|--zero
+        # -f|--force or -y|--yes may be required
 
-            # lvm has issues with persistence of metadata, so here comes the
-            # hammer...
+        # lvm has issues with persistence of metadata, so here comes the
+        # hammer...
+        self.exists = True
```

Comment:
Isn't it there to prevent ``pvcreate`` from having issues with existing metadata/format on the device? libblockdev's ``pvcreate`` doesn't use the ``-f`` option to force PV creation and so didn't blivet's ``pvcreate`` IIRC.